SpringProject/src/test/java/com/gradproject2019/UserServiceTest.java

```diff
@@ -108,6 +116,50 @@ public void shouldThrowErrorWhenNewEmailExists() {
         userService.editUser(token.getToken(), userId, update);
     }
 
+    @Test(expected = UserForbiddenException.class)
```

Comment:
Use the code coverage tool in intelliJ to ensure you've covered all branches with these tests (e.g. run all tests with coverage)

Reply:
then I'll approve - heavy handed hint but I'm but sure you're covering case 2 of your if statement